bundles/com.zeligsoft.domain.ngc.ccm.generator/src/com/zeligsoft/domain/ngc/ccm/generator/xtend/mainTransformAtcd.ext

```diff
@@ -218,7 +231,7 @@ create Specification mainTransformImplementation(MonolithicImplementation self)
 		self.interface.getComponentsHomes().processIncludes(this) ->
 		this.definitions.addAll(visit(definitions.flatten(), self.interface.eContainer));
 		
-emf::EObject copy(emf::EObject original, emf::EObject copyTo ) :
+emf::EObject copyEOb(emf::EObject original, emf::EObject copyTo ) :
```

Comment:
At line 216, new code calls `copyEObj`, but this method is called `copyEOb`. Is this a typo?